app/controllers/api/users_controller.rb

```diff
@@ -51,11 +51,13 @@ def user_params
   end
 
   def page
-    params[:page] || 1
+    page = params[:page] || 1
```

Comment:
remote the temp var and use `fetch` + cast it to int just after fetch: `my_hash.fetch(:digit, 123).to_i`